packages/@okta/vuepress-theme-prose/mixins/AnchorHistory.vue

```diff
@@ -51,14 +51,10 @@ export default {
       if (!target) {
         return;
       }
-      const scrollToPosition = target.offsetTop - this.getPaddedHeaderHeight();
-      window.scrollTo(0, scrollToPosition);
-      // Chrome & Safari: when zoomed in/out, window.scrollTo does not always perform scroll strictly equal to passed parameter
-      // https://bugs.chromium.org/p/chromium/issues/detail?id=890345
-      if (window.scrollY < scrollToPosition) {
```

Comment:
@emikhasyak-okta FYI, upon testing, my code changes functions correctly at standard zoom levels. Additionally, adding a scrollBy affects smooth scrolling behaviour.

Could you point me to any specific areas in the application where this change needs to be thoroughly tested?